tests/end2end_test/s3_ejp_xml_end_to_end_test.py

```diff
@@ -0,0 +1,62 @@
+import os
+import logging
+
+from data_pipeline.utils import get_yaml_file_as_dict
+from data_pipeline.dag_pipeline_config.xml_config import (
+    eJPXmlDataConfig
+)
+from dags.s3_xml_import_pipeline import (
+    DAG_ID,
+    named_literals
+)
+from dags.s3_xml_import_pipeline import (
+    DEFAULT_DEPLOYMENT_ENV_VALUE,
+    DEPLOYMENT_ENV_ENV_NAME
+)
+from tests.end2end_test import (
+    trigger_run_test_pipeline,
+    DataPipelineCloudResource
+)
+from tests.end2end_test.end_to_end_test_helper import (
+    AirflowAPI
+)
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def test_dag_runs_data_imported():
+    airflow_api = AirflowAPI()
+
+    data_pipeline_cloud_resource = (
+        get_data_pipeline_cloud_resource()
+    )
+    trigger_run_test_pipeline(
+        airflow_api,
+        DAG_ID,
+        data_pipeline_cloud_resource
+    )
+
+
+def get_data_pipeline_cloud_resource():
+    conf_file_path = os.getenv(
+        named_literals.EJP_XML_CONFIG_FILE_PATH_ENV_NAME
```

Comment:
Maybe we should use a separate environment variable for the test config. Especially since we are truncating the tables.

Reply:
maybe not, the configuration file used in the code here are just for testing in ci, and developement. configuration provided in the [formula](https://github.com/elifesciences/elife-data-hub-formula) is  used in the staging or production environment. and by the way, different configuration file is now used exclusively for `ci`, and so, one can actually change the destination table as one wishes.

Reply:
Just need to make sure it doesn't use the wrong one by accident.
e.g. I might configure it locally to test the pipeline and then run the end2end tests.

I guess the config here is read to know which tables to watch, but it is also read by the pipeline itself? (as part of the end2end test)